path_data.go

```diff
@@ -25,6 +25,34 @@ func matchAllNoTrailingSlashRegex(name string) string {
 // pathConfig returns the path configuration for CRUD operations on the backend
 // configuration.
 func pathData(b *versionedKVBackend) *framework.Path {
+	updateCreatePatchResponseSchema := map[int][]framework.Response{
+		http.StatusOK: {{
+			Description: "OK",
+			Fields: map[string]*framework.FieldSchema{
+				"version": {
+					Type:     framework.TypeInt64,
+					Required: true,
```

Comment:
Curious to know what required means in the context of a response? That it'll always be present in the payload even ifempty?

Reply:
Yes, we are using it to indicate that the field will always be present in a the given 200 response (as opposed to being conditionally present). `Required` is used in test validation to ensure that all the required fields are present.